pkg/zson/descriptor.go

```diff
@@ -6,6 +6,12 @@ import (
 	"github.com/mccanne/zq/pkg/zeek"
 )
 
+// MaxDescriptor is the largest descriptor ID allowed.  Since some tables
+// are sized based on the largest descriptor seen, very large descriptors
```

Comment:
very large descriptors -> very large numbers of descriptors?

Reply:
no, I'm thinking about large garbage descriptors where you have 1, 2, 3, then 2^27 because of a bug or bad input data.   It's set to 1,000,000 so we can handle very large numbers of descriptors but not run out of memory in the face of bugs.

Reply:
ah, i see. I thought "very large descriptors" meant "very wide descriptors", but it means "very large descriptor IDs"